bin/public/hardening/proxy.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+'''
+Setup proxy function
+
+'''
+
+from general import x
+import app
+from scopen import scOpen
+
+
+def install_proxy_setup():
+    x("cp %shardening/proxy.sh /etc/profile.d" % app.SYCO_VAR_PATH)
+    x("chmod 644 /etc/profile.d/proxy.sh")
+    #Pass proxy variable when running sudo
+    proxenv='Defaults env_keep +="http_proxy"'
+    x("echo "+proxenv+" >> /etc/sudoers")
```

Comment:
This will create duplicates the second time it's run and forward. Should either use augeas or some other means of only adding it to the file if it doesn't exist. Hardening specifically should be run often so this problem will grow.